collection-scripts/gather

```diff
@@ -4,7 +4,7 @@
 resources=()
 
 # KubeVirt HCO related namespaces
-resources+=(ns/kubevirt-hyperconverged ns/openshift-operator-lifecycle-manager ns/openshift-marketplace)
+resources+=(ns/openshift-cnv ns/kubevirt-hyperconverged ns/openshift-operator-lifecycle-manager ns/openshift-marketplace)
```

Comment:
I'd rather having a list of namespaces supported by cnv and a list supported by kubevirtci and maintain different images for each instead where each image contains the expected namespaces to collect.

I"ll ack this patch since the failure in case of a missing namespace isn't severe
`Error: namespaces "kubevirt-hyperconverged" not found`
and i'll look for a better approach of that.